ui/app/components/clients/chart-container.hbs

```diff
@@ -3,49 +3,46 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{! HBS display template for rendering title, description and stat boxes with a chart on the right }}
+{{! Display template for rendering title, description and stat boxes with a chart on the right and legend below }}
 
-<div class="chart-wrapper single-chart-grid" data-test-chart-container={{@title}} ...attributes>
```

Comment:
Since I changed the wrapping element, recommend viewing with ✔️ hide white space